controllers/comments.controllers.js

```diff
@@ -12,3 +15,17 @@ exports.getCommentsByArticleId = (req, res, next) => {
 		})
 		.catch(next);
 };
+
+exports.postCommentsByArticleId = (req, res, next) => {
+	const { article_id } = req.params;
+	const insert = req.body;
+
+	selectArticleById(article_id)
+		.then(() => {
+			return insertCommentsByArticleId(article_id, insert);
```

Comment:
I like the idea behind this, but these two promises can happen at the same time and this should still work as expected.